pkg/porter/explain_test.go

```diff
@@ -210,7 +210,7 @@ func TestExplain_generatePrintableBundleParams(t *testing.T) {
 		},
 	}
 
-	pb, err := generatePrintable(bun)
+	pb, err := generatePrintable(bun, "")
```

Comment:
Right now the existing tests aren't enough to validate your change. Let's copy this test `TestExplain_generatePrintableBundleParams` and change it so that one of the parameters has `ApplyTo` set to "install". Then we can check that when we provided "install" it shows up and when we provide "upgrade" as the action, it is not present in the generated parameters.